deployment/ds-hostnet-split/03-envoy.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         - node0
         command:
         - envoy
-        image: docker.io/envoyproxy/envoy-alpine:v1.9.0
+        image: docker.io/envoyproxy/envoy:v1.9.1
```

Comment:
any reason why you're switching from alpine to ubuntu based envoy image for the hostnet deployment?